snapm/manager/plugins/stratis.py

```
@@ -999,6 +1042,11 @@ def revert_snapshot(self, name):
             raise SnapmPluginError(
                 f"Unexpected D-Bus error setting property for {pool_name}/{fs_name}"
             ) from err
+        except DbusClientMissingPropertyError as err:
+            self._log_warn("Could not access filesystem.MergeScheduled property")
+            raise SnapmPluginError(
+                f"Unexpected error accessing filesystem.MergeScheduled property"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)